docs/source/upcoming_release_notes/1095-lodcm_energy.rst

```diff
@@ -0,0 +1,30 @@
+1095 lodcm_energy
+#################
+
+API Changes
+-----------
+- Add energy setpoint and tweakX to the LODCM class
+
+Features
+--------
+- Set energy directly
```

Comment:
This will be rendered in the documentation as such: `Set energy directly` - so it  needs context saying it's LODCM-related